ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/generate-default-config.yml

```diff
@@ -1,3 +1,4 @@
-generate_default_config: false
+generate_default_config: true
```

Comment:
The purpose of this test case was to test the feature knob `generate_default_config: false` showing an empty config. Please create a separate test case for transceiver....

Reply:
transceiver_qsfp_default_mode_4x10_false